bootstrap.sh

```diff
@@ -560,17 +561,29 @@ install_brewfile "$SENTRY_ROOT"
 setup_pyenv "$SENTRY_ROOT"
 # Run it here to make sure pyenv's Python is selected
 eval "$(pyenv init --path)"
+# shellcheck disable=SC2155
+export PYENV_VERSION=$(
+  # shellcheck disable=SC1090 disable=SC1091
+  source "${SENTRY_ROOT}/scripts/lib.sh"
+  get-pyenv-version
+)
```

Comment:
It kind of sucks that there's so many little steps in here. In the future, I will look into unifying things into one single Python target.

Reply:
Since we don't have a `.python-version` file, we need to tell `pyenv` via this variable what Python version to load. If we don't do this, the `setup_virtualenv` will fail.

I'm filing [this][1] to improve this situation.

[1]: https://github.com/getsentry/sentry/issues/28867

Reply:
I could get rid of this step if I installed `direnv` and I run `direnv allow`.